### ISSUE TITLE
I can't get the projects to load correctly. This is the error message I'm getting in Visual Studios C++ 2010.

### ISSUE THREAD

**Issue by nateEE** [reed-adept/aria-matlab#1]
C:\Program Files\MobileRobots\Aria\matlab\aria-matlab-master\ariac-dll-vc2010.vcxproj : warning  : Platform 'x64' referenced in the project file 'ariac-dll' cannot be found. Please make sure you have it installed under '%VCTargetsPath%\Platforms\x64'.

C:\Program Files\MobileRobots\Aria\matlab\src\AriaDLL-vc2010.vcxproj : error  : Project "C:\Program Files\MobileRobots\Aria\matlab\src\AriaDLL-vc2010.vcxproj" could not be found.

C:\Program Files\MobileRobots\Aria\matlab\aria-matlab-master\ariac-test-vc2010.vcxproj : warning  : Platform 'x64' referenced in the project file 'ariac-test-vc2010' cannot be found. Please make sure you have it installed under '%VCTargetsPath%\Platforms\x64'.

**Comment by reed-adept** [comment]
I think you can solve the error (second one) by moving the contents of aria-matlab-master to C:\Program Files\MobileRobotAria\matlab.  IN other words, the contents of the repository sholuld be in the "matlab" directory.

(Or, to build outside of the Aria directory, replace the AriaDLL project in the ariac solution with AriaDLL from \Program Files\MobileRobots\Aria.

I think you can ignore the warnings about x64 if you are on a 32-bit system and don't have the Platform SDK installed.

**Comment by nateEE** [comment]
Thank you for your help. I am trying to use this for my Master's Thesis. Im a long time Matlab user but new to C++, Visual Studios, and Mex files. I am now having problems creating the mex files. I'm getting this error.
________________________________________________________________________________________-
   Creating library C:\Users\Nate\AppData\Local\Temp\mex_xqATHF\templib.x and object C:\Users\Nate\AppData\Local\Temp\mex_xqATHF\templib.exp 
aria_init.obj : error LNK2019: unresolved external symbol __imp_arrobot_setpose referenced in function arrobot_setpose_p 
aria_init.obj : error LNK2019: unresolved external symbol __imp_arrobot_setwheelvels referenced in function arrobot_setvel2 
aria_init.obj : error LNK2019: unresolved external symbol __imp_aria_init referenced in function mexFunction 
aria_init.obj : error LNK2019: unresolved external symbol __imp_aria_setloghandler referenced in function mexFunction 
aria_init.mexw64 : fatal error LNK1120: 4 unresolved externals 

  C:\PROGRA~1\MATLAB\R2012B\BIN\MEX.PL: Error: Link of 'aria_init.mexw64' failed. 

---

I changed some of the variable cmd because of another error I was receiving.

---

  C:\PROGRA~1\MATLAB\R2012B\BIN\MEX.PL: Error: 'mex-src/aria_init.c' not found. 

---

I changed the variable cmd to:

mex -g -DWIN32 -DMATLAB -IC:/Progra~1/MobileRobots/ARIA/matlab 
-LC:/Progra~1/MobileRobots/ARIA/matlab -lariac_vc10_x64 
-LC:/Progra~1/MobileRobots/ARIA/lib -lAriaVC10 
C:/Progra~1/MobileRobots/ARIA/matlab/mex-src/aria_init.c

these are the paths to ariac.h, ariac_vc_x64.lib, ariavc10.lib, and aria_init.c

I am getting this error with the makemex.m file without changing the variable cmd:

---

  C:\PROGRA~1\MATLAB\R2012B\BIN\MEX.PL: Error: 'mex-src/aria_init.c' not found. 

---

The mexw64 file is outputting into this directory. Not sure if this matters.
C:\Users\Nate\Documents\MATLAB\aria_init.mexw64

Thank you for you help again.

**Comment by reed-adept** [comment]
Does it work if you change to (open) the C:\Program Files\MobileRobots\Aria\matlab directory in Matlab so that it is the working directory?

At some point I would like to make it easier to use outside of C:\Program Files since that’s a system directory and Windows 7+ does odd things if you are not an Administrator (which would be another thing you could try) but haven’t had a chance yet.
